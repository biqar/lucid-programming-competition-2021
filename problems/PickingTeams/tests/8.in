100
+ m l
? n k
? h e
+ i b
+ e m
+ l d
+ l j
+ h m
+ l j
+ c k
+ f l
+ k e
+ f j
+ c k
? n g
+ f j
+ l a
+ i j
? e i
+ g o
? b c
+ o e
+ h j
+ h k
? f n
+ b g
? m n
? n d
? n k
+ m b
+ i m
+ a k
+ j e
+ a l
? d h
+ k f
+ d l
? j c
+ k i
? n e
+ d a
? n c
+ j c
+ k c
? h f
+ m l
+ n m
+ b k
? e i
+ h e
? l a
+ j d
? k m
? g h
+ b i
+ b m
+ o c
? j b
+ n e
+ g m
+ o h
+ n g
+ e n
+ a b
+ m g
+ a o
+ e d
? n m
? k f
+ d j
+ d f
? j o
+ l g
+ c b
+ b h
+ e c
? k g
? d j
+ d k
+ g b
+ m j
? o g
+ m a
+ c e
+ j k
+ n e
+ h b
+ d b
+ h l
+ h f
? i m
? i d
? n l
+ b l
+ l h
+ o e
+ g h
? l e
+ o n
+ a n
