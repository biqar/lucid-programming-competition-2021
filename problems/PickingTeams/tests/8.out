no
no
no
no
no
no
no
no
no
no
no
no
no
yes
no
yes
yes
yes
no
no
yes
no
yes
yes
no
yes
no
no
no
5
